saleor/graphql/order/types.py

```diff
@@ -1200,14 +1201,47 @@ class Order(ModelObjectType):
         + ADDED_IN_38
         + PREVIEW_FEATURE,
     )
-    total_pending_refund = PermissionsField(
```

Comment:
We shouldn't rename fields like that. this one should be deprecated and a new one should be added.